extensions/applicationinsights-cfgsync-js/src/Interfaces/ICfgSyncConfig.ts

```diff
@@ -18,6 +18,12 @@ export interface ICfgSyncConfig {
      * If set, any core config changes will be ignored.
      */
     cfgUrl?: string;
+
+    /**
+     * Identifies if fetching cdn endpoint should be blocked
+     * Default to false.
+     */
+    blockCdn?: boolean;
```

Comment:
added `blockCdn` config here. Default: `false`. 
If it is set to `true`, fetch/xhr will not make fetching calls

Reply:
TODO:
currently `blockCdn` is inside `onConfigChange` so it will listen to `core.config` changes.
need to do: (haven't added)
1. true -> false: clear timer
2. false -> true trigger fetch and timer